<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_head</class>
 <widget class="QWidget" name="menu_head">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>101</width>
    <height>166</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="menu_head_img">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/theventi/D104.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="menu_head_name">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>저당 바닐라라떼</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="menu_head_price">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4,800원</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
